Lib/Configurations/Base.xcconfig

```diff
@@ -38,4 +44,5 @@ WATCHOS_DEPLOYMENT_TARGET = 2.0;
 TVOS_DEPLOYMENT_TARGET = 9.0;
 MACOSX_DEPLOYMENT_TARGET = 10.9;
 
-SWIFT_VERSION = 3.0;
+SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
This should be set to `Default` instead of `On`. It now produces a warning (mode deprecated).

Reply:
Yup, thanks.